model/registry/vcs.yaml

```diff
@@ -5,6 +5,17 @@ groups:
       This group defines the attributes for
       [Version Control Systems (VCS)](https://en.wikipedia.org/wiki/Version_control).
     attributes:
+      - id: vcs.repository.name
```

Comment:
`vcs.repository.name` was added since it was used by the githubreceiver.
Should this be changed to one of the existing attributes in the registry (eg. `vcs.repository.url.full`)?

Reply:
Applications could calculate `vcs.repository.name` from `vcs.repository.url.full`. It is worth adding a new field (space) to avoid the calculation.

In the case of Github, for example:

```yaml
vcs.repository.name: open-telemetry/semantic-conventions
vcs.repository.url.full: https://github.com/open-telemetry/semantic-conventions
```

I chose this repo because is an actual repo. We will add 35 bytes, to each change document, it is not much. In this example repo, we have 1385 PRs so `1385*35~=47K`.

Reply:
On the original PR for VCS attributes, there was discussion about this value and it was determined that `url.full` was the preferred path for that iteration. 

In the GitHub example, `open-telemetry/semantic-conventions` (<org>/<repo name>) is actually considered the `full_name` in queries. IIRC the same holds true for GitLab except it can be longer due to how things are pathed in GitLab (<group>/<subgroup>/..../<repo/project name>).

Unless anyone feels strongly, I'd opt for updating the GitHub receiver to use `vcs.repository.url.full` in place of `vcs.repository.name`. The reason I didn't do it is because it requires a query code change.

Reply:
In 41543cdc9d4d76f9327d8689acfacc86c70c637a I replaced the use of `vcs.repository.name` with `vcs.repository.url.full`.
@adrielp Shall we create a separate issue to discuss a future addition of `vcs.repository.name`?

Reply:
Sure, we can open a piece of work to track it for later.

Reply:
Opened #1453